tests/testthat/test_evaluateStandard.R

```diff
@@ -21,15 +21,15 @@ test_that("a list with the expected properties and structure is returned",{
 
 test_that("expected number of checks (in)valid",{
   expect_equal(evaluateStandard(data=adlbc, standard="sdtm")[["valid_count"]],1)
-  expect_equal(evaluateStandard(data=adlbc, standard="sdtm")[["invalid_count"]],9)
+  expect_equal(evaluateStandard(data=adlbc, standard="sdtm")[["invalid_count"]],10)
   
   adlbc_edit <- adlbc
   adlbc_edit$TEST <- adlbc_edit$PARAM
   a<-evaluateStandard(data=adlbc_edit, standard="sdtm")
   expect_equal(a[["valid_count"]],2)
-  expect_equal(a[["invalid_count"]],8)
-  expect_equal(a[["total_count"]],10)
-  expect_equal(a[["match_percent"]],.2)
+  expect_equal(a[["invalid_count"]],9)
+  expect_equal(a[["total_count"]],11)
+  expect_equal(a[["match_percent"]],.182)
```

Comment:
this still does not work - it appears as if it's off by -0.000182 so probs just need to add a round function